Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builder image
-FROM adoptopenjdk:15_36-jdk-hotspot as build
```

Comment:
using this pr to bump to java 16 as well

Reply:
Can we update the references to the Java version in the docs? E.g. readme, GitHub Pages.